pkg/registry/service/strategy.go

```diff
@@ -118,3 +118,22 @@ func MatchServices(label labels.Selector, field fields.Selector) generic.Matcher
 func ServiceToSelectableFields(service *api.Service) fields.Set {
 	return generic.ObjectMetaFieldsSet(service.ObjectMeta, true)
 }
+
+type serviceStatusStrategy struct {
+	svcStrategy
+}
+
+var StatusStrategy = serviceStatusStrategy{Strategy}
```

Comment:
please add a godoc for exported variable:
// StatusStrategy is the default logic invoked when creating and updating Services.